OutOfSchool/OutOfSchool.DataAccess/Models/Workshop.cs

```diff
@@ -79,6 +79,8 @@ public class Workshop
         [MaxLength(60)]
         public string ProviderTitle { get; set; } = string.Empty;
 
+        public string Keywords { get; set; } = string.Empty;
```

Comment:
It would be nice to limit string field

Reply:
Dear @SergeyNovitsky,
Thank you for your advice. This was done. Please check.